firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/CommonUtils.java

```diff
@@ -525,6 +527,15 @@ public static List<BuildIdInfo> getBuildIdInfo(Context context) {
     return buildIdInfoList;
   }
 
+  @Nullable
+  public static String getVersionControlInfo(Context context) {
+    int id = getResourcesIdentifier(context, VERSION_CONTROL_INFO_RESOURCE_NAME, "string");
```

Comment:
Is it possible id is null?

Reply:
`int` cannot be null, this will return 0 when the android resource is not found.